### ISSUE TITLE
Notifications option does not save setting.

### ISSUE THREAD

**Issue by l300lvl** [brianrobles204/Music-Integration#16]
I thought there was a bug for this but since I don't see it:

When setting the Preferences>Notification option to Off, the setting is only temporary and is reset to the On status whenever the shell is restarted or the system reboot, etc, and the message tray indicator is indeed back from the grave. It isn't the end of the world, but the setting is useless in the current state.

**Comment by l300lvl** [comment]
I actually discovered that neither option, Notifications, OR Cover Art Time Overlay, save their settings on restart.

I will try to reproduce this issue and check my schema as well, just to verify.

**Comment by brianrobles204** [comment]
Damn, I can reproduce this too. Although, the music placement option is still working, some other options are broken. You should know that the extension on this github page is still in beta; you can grab the stable version from https://extensions.gnome.org/extension/30/music-integration/ I shall investigate, but for now you may as well stay put and use the stable if you want.

**Comment by l300lvl** [comment]
For the last 48 hours or so I discovered the ability to use the settings in the metadata, which actually do work. I posted that on E.G.O. Sorry I didn't see that before. I am using the version from there but I will watch for updates. Thanks for the quick response!

**Comment by brianrobles204** [comment]
Yes, I added that as backup :) I'll try to fix the actual problem itself as soon as my busy schedule lets me~ Thanks for using my extension

**Comment by semente** [comment]
I use the version from extensions.gnome.org and I have the same problem.

Looking forward for a fix. :-)

Thanks!
